mapproxy/cache/geopackage.py

```diff
@@ -0,0 +1,657 @@
+                      matrix_height INTEGER NOT NULL, -- Number of rows (>= 1) in tile matrix at this zoom level
+                      tile_width    INTEGER NOT NULL, -- Tile width in pixels (>= 1) for this zoom level
+                      tile_height   INTEGER NOT NULL, -- Tile height in pixels (>= 1) for this zoom level
+                      pixel_x_size  DOUBLE  NOT NULL, -- In t_table_name srid units or default meters for srid 0 (>0)
+                      pixel_y_size  DOUBLE  NOT NULL, -- In t_table_name srid units or default meters for srid 0 (>0)
+                      CONSTRAINT pk_ttm PRIMARY KEY (table_name, zoom_level), CONSTRAINT fk_tmm_table_name FOREIGN KEY (table_name) REFERENCES gpkg_contents(table_name))
+                  """,
+                 """
+                         CREATE TABLE IF NOT EXISTS gpkg_tile_matrix_set
+                 (table_name TEXT    NOT NULL PRIMARY KEY, -- Tile Pyramid User Data Table Name
+                  srs_id     INTEGER NOT NULL,             -- Spatial Reference System ID: gpkg_spatial_ref_sys.srs_id
+                  min_x      DOUBLE  NOT NULL,             -- Bounding box minimum easting or longitude for all content in table_name
+                  min_y      DOUBLE  NOT NULL,             -- Bounding box minimum northing or latitude for all content in table_name
+                  max_x      DOUBLE  NOT NULL,             -- Bounding box maximum easting or longitude for all content in table_name
+                  max_y      DOUBLE  NOT NULL,             -- Bounding box maximum northing or latitude for all content in table_name
+                  CONSTRAINT fk_gtms_table_name FOREIGN KEY (table_name) REFERENCES gpkg_contents(table_name), CONSTRAINT fk_gtms_srs FOREIGN KEY (srs_id) REFERENCES gpkg_spatial_ref_sys (srs_id))
+                  """,
+                 """
+                 CREATE TABLE IF NOT EXISTS {0}
+                    (id          INTEGER PRIMARY KEY AUTOINCREMENT, -- Autoincrement primary key
+                     zoom_level  INTEGER NOT NULL,                  -- min(zoom_level) <= zoom_level <= max(zoom_level) for t_table_name
+                     tile_column INTEGER NOT NULL,                  -- 0 to tile_matrix matrix_width - 1
+                     tile_row    INTEGER NOT NULL,                  -- 0 to tile_matrix matrix_height - 1
+                     tile_data   BLOB    NOT NULL,                  -- Of an image MIME type specified in clauses Tile Encoding PNG, Tile Encoding JPEG, Tile Encoding WEBP
+                     UNIQUE (zoom_level, tile_column, tile_row))
+                  """.format(self.table_name)
+                 ]
+
+        for stmt in stmts:
+            db.execute(stmt)
+
+        db.execute("PRAGMA foreign_keys = 1;")
+
+        # List of WKT execute statements and data.
+        wkts = [("""
+                INSERT OR REPLACE INTO gpkg_spatial_ref_sys (
```

Comment:
Can you refactor this SQL statement? The same INSERT INTO lines are repeated 5 times.